### PR TITLE
SelectorExecutor class has been added into client driver's embedding utils

### PR DESCRIPTION
**Pull request by arubtsov**

### CODE REVIEW

File: src/test-run/execute-js-expression.js [hunk 1]
```diff
@@ -2,16 +2,19 @@ import { createContext, runInContext } from 'vm';
 import SelectorBuilder from '../client-functions/selectors/selector-builder';
 import ClientFunctionBuilder from '../client-functions/client-function-builder';
 
-export function executeJsExpression (expression, skipVisibilityCheck, testRun) {
-    var sandbox = {
+export function executeJsExpression (expression, skipVisibilityCheck, testRun, collectionMode) {
```
Comment (churkin) [inline src/test-run/execute-js-expression.js]:
```javascript
export function executeJsExpression (expression, testRun, { skipVisibilityCheck, collectionMode }) {
```
?

**Comment by arubtsov** [comment]
@testcafe-build-bot retest

**Comment by arubtsov** [comment]
@testcafe-build-bot retest

**Comment by arubtsov** [comment]
FPR

**Comment by kirovboris** [comment]
ping @churkin

**Pull request merged**